site/gulpfile.js

```diff
@@ -20,42 +28,100 @@ gulp.task('pug', () => {
             pretty: true
         }))
         .pipe(gulp.dest('dist/'));
-})
+});
 
 gulp.task('styl', () => {
     return gulp.src('src/styl/index.styl')
         .pipe(stylus({
             compress: true
         }))
         .pipe(gulp.dest('dist/assets/css'));
-})
+});
 
 gulp.task('js', () => {
     return gulp.src('src/js/*.js')
         .pipe(concat('index.js'))
         .pipe(babel({
-            "presets": [
+            'presets': [
```

Comment:
Since this is an object quotes aren't required at all

Reply:
Fixed